ledgereth/__main__.py

```diff
@@ -86,9 +86,24 @@ def get_args(argv):
         "-p",
         "--gasprice",
         type=int,
-        required=True,
+        required=False,
         help="The gas price to use for the tx",
     )
+    send_parser.add_argument(
```

Comment:
Maybe a good TODO for the backlog is to convert this to use the `click` framework?

Reply:
Nice thing about click... we can just alias this CLI directly into `ape` as a subcommand by re-importing/referencing the CLI in the plugin's `setup.py` (see `tokenlists` CLI)

Reply:
I've always just used ArgumentParser and done some pretty complex stuff with it.  Haven't checked out click before, but will do.  Should probably note that the CLI functionality of this package was never really meant to be a full feature but just a nice quick way to test the library.  

Making transactions on the CLI is kinda clunky:

    python -m ledgereth send [LEDGER_ACCOUNT] [DESTINATION] 100000000000000000 -n 4 -p 50100000000 -c 137

I'm pretty open to other options though, but would probably want to flesh out more of a UX with useful functionality and that's probably out of scope of this PR.